eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/BackPropagatedRSWriter.java

```diff
@@ -96,6 +103,12 @@ public void write(
                         // Don't bring such variants in for clustering again.
                         .filter(SubmittedVariantEntity::isAllelesMatch)
                         .map(AccessionedDocument::getAccession).collect(Collectors.toList());
+
+        // There may be some unclustered SS left behind which could potentially be assigned
+        // an existing RS in the current assembly
```

Comment:
Not so important but it might be clearer to continue to refer to this as the "original assembly", both here and in the following code.